src/AdoNetCore.AseClient/Internal/Handler/StreamingDataReaderTokenHandler.cs

```diff
@@ -61,9 +63,16 @@ public void Handle(IToken token)
                         Items = row.Values
                     });
                     break;
-                case DoneToken _:
-                case DoneInProcToken _:
-                case DoneProcToken _:
+                case DoneToken t:
```

Comment:
Can these cases be compacted down like they were before, with a single call to `EnsureResultExists`?